steps/start/postcode-checker/BranchForEnglandOrWales.js

```diff
@@ -0,0 +1,64 @@
+const { branch, goTo } = require('@hmcts/one-per-page');
+const request = require('request');
+const config = require('config');
+const { Logger } = require('@hmcts/nodejs-logging');
+
+const postcodeCountryLookupUrl = config.get('postcodeChecker.url');
+const postcodeCountryLookupToken = config.get('postcodeChecker.token');
+const postcodeCountryLookupAllowedCountries = config.get('postcodeChecker.allowedCountries');
+
+const logger = Logger.getLogger('PostcodeChecker.js');
+
+class BranchForEnglandOrWales {
+  constructor(postcode, englandOrWalesStep, otherStep, errorStep) {
+    this.postcode = postcode;
+    this.englandOrWalesStep = englandOrWalesStep;
+    this.otherStep = otherStep;
+    this.errorStep = errorStep;
+  }
+
+  isEnglandOrWalesPostcode() {
+    const postcode = this.postcode;
+    return new Promise((resolve, reject) => {
+      request.get({
```

Comment:
We're using superagent for CYA (and other projects in SYA). Is it possible to make this work with that instead of request for consistency?